thinc/backends/numpy_ops.pyx

```diff
@@ -55,7 +55,7 @@ class NumpyOps(Ops):
         else:
             return self.xp.array(data)
 
-    def allocate(self, shape, dtype='float32'):
+    def alloc(self, shape, dtype='float32'):
         if isinstance(shape, int):
```

Comment:
Let's kill this `isinstance` condition? I think it was a hold-over from oldthinc.